src/target/stm32l4.c

```diff
@@ -28,44 +28,49 @@
  * RM0394 STM32L43xxx STM32L44xxx STM32L45xxx STM32L46xxxx advanced
  *  ARM®-based 32-bit MCUs Rev.3
  * RM0432 STM32L4Rxxx and STM32L4Sxxx advanced Arm®-based 32-bit MCU. Rev 1
+ * RM0434 STM32WB55xx Rev 2 (pre production) and Rev 4 (final HW)
  *
  *
  */
 
 #include "general.h"
 #include "target.h"
+#include "command.h"
 #include "target_internal.h"
 #include "cortexm.h"
 
-static bool stm32l4_cmd_erase_mass(target *t);
```

Comment:
Unneeded shuffling around unchanged code.
\

Reply:
Yes, I had moved that without noticing and forgot to reset the position, no reason to do that.